autode/neb/original.py

```diff
@@ -204,6 +206,23 @@ def __init__(self, num):
 
 class NEB:
 
+    def plot_surface(self, name='neb'):
```

Comment:
Would it be possible to do some sort of 2d plotting of the PES?

Reply:
not easily

Reply:
fair, I guess it wouldn't be worth it without a 2d pes to put it on anyway